src/uwtools/logging.py

```diff
@@ -16,6 +17,27 @@
 MSGWIDTH = 69
 
 
+class _Logger:
+    """
+    Support for swappable loggers.
+    """
+
+    def __init__(self) -> None:
+        self.logger = logging.getLogger()  # default to Python root logger.
```

Comment:
When a `uwtools.logging._Logger` object is instantiated, it will initially be connected to the Python root logger, which is what we want in CLI mode.